code/game/shuttles/vox.dm

```diff
@@ -62,6 +62,10 @@ var/global/datum/shuttle/vox/vox_shuttle = new(starting_area=/area/shuttle/vox/s
 	link_to(vox_shuttle)
 	.=..()
 
+/obj/machinery/computer/shuttle_control/vox/attack_ai(mob/user)
+	to_chat(user, "<span class='warning'>This computer has a firewall preventing you from interacting with it.</span>")
```

Comment:
Missed opportunity to add in a funny message like "You can't make sense of this terminal's programming, it's all chicken scratch."